dash_bio/Ideogram.py

```diff
@@ -155,7 +155,9 @@ class Ideogram(Component):
 cytogenetically well-characterized organisms, e.g. human, beside chromosomes of
 less studied organisms, e.g. chimpanzee.
 - showNonNuclearChromosomes (boolean; optional): Whether to show non-nuclear chromosomes,
-e.g. for mitochondrial (MT) and chloroplast (CP) DNA."""
+e.g. for mitochondrial (MT) and chloroplast (CP) DNA.
+
+Available events: """
```

Comment:
This was generated using the wrong version of Dash -- please use `0.37.0` or higher!